client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -788,6 +790,11 @@ export class BpmnEditor extends CachedComponent {
           engineProfile={ engineProfile }
           onChange={ (engineProfile) => this.engineProfile.set(engineProfile) } />
         }
+
+        <VariableTab
+          id={ this.props.id }
+          layout={ layout }
+          injector={ this.getModeler().get('injector') } />
```

Comment:
Perhaps extract `injector` to a variable like `imported` is in this scope.

Reply:
Adjusted in [da0ba48](https://github.com/camunda/camunda-modeler/pull/4459/commits/da0ba489142caa437e38f48d5cb2e9d2e0281784)